src/components/CustomCollection/CustomCollection.tsx

```diff
@@ -242,13 +246,18 @@ const CustomCollection = ({
   return (
     <>
       <Collection header={customCollectionHeader} footer={addLinkForm}>
-        {bookmarks.map((bookmark: BookmarkType) => (
-          <RemovableBookmark
-            key={`bookmark_${bookmark._id}`}
-            bookmark={bookmark}
-            handleRemove={() => handleRemoveBookmark(`${bookmark._id}`)}
-          />
-        ))}
+        {bookmarks.map(
```

Comment:
minor nit: we should use `.filter().map()` since Array.map is meant to be used if each item is always going to return an item. ie:

```
{bookmarks.filter(b: BookmarkType => !b.isRemoved).map(...
```

Reply:
thank you!